pkg/util/validator/validator.go

```diff
@@ -29,6 +30,18 @@ func Struct(s interface{}) []error {
 	return nil
 }
 
+func StructAllError(s interface{}) error {
+	var errMsg string
+	allErrs := Struct(s)
+	if len(allErrs) > 0 {
+		for order, err := range allErrs {
+			errMsg += fmt.Sprintf("Error[%d]: %s", order, err)
```

Comment:
missing `\n`

Reply:
This operation is not necessary. `go-playground/validator` itself returns an error, and I expanded the error to `[]error` in `Struct` func.
So this function just needs `v.Struct()`

validator error: https://github.com/go-playground/validator/blob/master/errors.go#L35-L57

example:
```go
type Options struct {
	Field1 string `validate:"required"`
	Field2 string `validate:"required"`
}

func main() {
	opts := Options{}
	fmt.Println(validator.New().Struct(&opts))
	// Output:
	// Key: 'Options.Field1' Error:Field validation for 'Field1' failed on the 'required' tag
	// Key: 'Options.Field2' Error:Field validation for 'Field2' failed on the 'required' tag
}
```